scaladoc/src/dotty/tools/scaladoc/renderers/HtmlRenderer.scala

```diff
@@ -158,7 +167,7 @@ class HtmlRenderer(rootPackage: Member, val members: Map[DRI, Member])(using ctx
           val attr = if expanded || isSelected then Seq(cls := "expanded") else Nil
           (isSelected || expanded) -> div(attr)(
             linkHtml(expanded),
-            span(),
+            span(cls := "ar"),
```

Comment:
What `ar` name comes from? Maybe we would like to name it more meangingful?

Reply:
arrow. We will be adding tons of this to all pages so we want to keep it as short as possible